### PR TITLE
Feat: adv_core/adv_dhcp_server - Add multiple features

### PR DESCRIPTION
**Pull request by oxedions**
Add multiple features. See CHANGELOG.

Also greatly improve role rendering time.

### CODE REVIEW

**Comment by oxedions** [comment]
Note: CHANGELOG will be sorted at merge, since it was sorted in another PR.

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,9 +16,14 @@
   - addons/nic_nmcli:
     - convert to new inventory format (#401)
     - add all ansible nmcli module capabilities (#444)
+  - advanced_core/advanced_dhcp_server:
+    - add multiple entry per host capability
```
Comment (bobitame) [inline CHANGELOG.md:20]: entry -> entries

File: roles/advanced-core/advanced_dhcp_server/readme.rst [hunk 2]
```diff
@@ -57,21 +85,98 @@ And ice1-2:
...
 
-**opt 61 and opt 82**
-"""""""""""""""""""""
+opt 61 and opt 82
+"""""""""""""""""
 
-It is possible to use advanced dhcp features to identify an host. The following parameters are available, for the host and its BMC. Note that only one of these must be set for an host/BMC at the same time:
+It is possible to use advanced dhcp features to identify an host. The following
+parameters are available, for the host and its BMC. Note that only one of these
+must be set for an host/BMC at the same time:
```
Comment (bobitame) [inline roles/advanced-core/advanced_dhcp_server/readme.rst:93]: this note is no longer valid as the "multiple entries per host" functionality is provided by this PR

File: roles/advanced-core/advanced_dhcp_server/tasks/main.yml [hunk 3]
```diff
@@ -44,7 +44,7 @@
 - name: Template >> /etc/dhcp/dhcpd.conf
   template:
     src: dhcpd.conf.j2
-    dest: /etc/dhcp/dhcpd.conf
+    dest: /etc/dhcp/dhcpd.conf.new
```
Comment (osmocl) [inline roles/advanced-core/advanced_dhcp_server/tasks/main.yml:47]: The unitary tests are failing due to ` .new` suffix added in all configuration files ?
Why renaming all configuration files ?

Also the following configuration will fail:

```
# cat /etc/dhcp/dhcpd.conf.new
[...]

## NETWORKS AND NODES

include "/etc/dhcp/dhcpd.networks.conf";
```

Comment (oxedions) [reply roles/advanced-core/advanced_dhcp_server/tasks/main.yml:47]: Oups, thanks a lot !
This is a bug. Correcting.

File: roles/advanced-core/advanced_dhcp_server/readme.rst (hunk 2 above)
Comment (oxedions) [reply roles/advanced-core/advanced_dhcp_server/readme.rst:93]: You are right, I will correct it. Thanks a lot!

File: CHANGELOG.md (hunk 1 above)
Comment (oxedions) [reply CHANGELOG.md:20]: Good catch!

**Pull request merged**